contracts/storage/RibbonStakingVaultStorage.sol

```diff
@@ -1,12 +1,46 @@
 // SPDX-License-Identifier: MIT
 pragma solidity =0.8.4;
 
+import {Vault} from "../libraries/Vault.sol";
+
 abstract contract RibbonStakingVaultStorageV1 {}
 
 // We are following Compound's method of upgrading new contract implementations
 // When we need to add new storage variables, we create a new version of RibbonThetaVaultStorage
 // e.g. RibbonThetaVaultStorage<versionNumber>, so finally it would look like
 // contract RibbonThetaVaultStorage is RibbonThetaVaultStorageV1, RibbonThetaVaultStorageV2
 abstract contract RibbonStakingVaultStorage is RibbonStakingVaultStorageV1 {
+  /// @notice Stores the user's pending deposit for the round
```

Comment:
We want to move these variables into RibbonStakingVaultStorageV1 so the next upgrade can be done by inheriting RibbonStakingVaultStorageV2

Reply:
u right. switched